src/Routes/Signatures/Signatures.scss

```diff
@@ -0,0 +1,3 @@
+.sigTable {
+    width: 100%;
+}
```

Comment:
I _think_ you can take this out and the table will still fill the full width of the card

Reply:
YUP now its gooooodddd